src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/compiled_model/import_export.cpp

```diff
@@ -27,17 +25,25 @@ const std::vector<ov::element::Type_t> ovExecGraphInfoElemTypes = {
         ov::element::bf16,
         ov::element::boolean,
 };
-
 INSTANTIATE_TEST_SUITE_P(ov_compiled_model,
                          OVCompiledGraphImportExportTest,
                          ::testing::Combine(
                                  ::testing::ValuesIn(ovExecGraphInfoElemTypes),
                                  ::testing::Values(targetDevice),
                                  ::testing::Values(pluginConfig)),
                          OVCompiledGraphImportExportTest::getTestCaseName);
-
 INSTANTIATE_TEST_SUITE_P(
         ov_compiled_model, OVClassCompiledModelImportExportTestP,
         ::testing::Values(targetDevice));
 
-}  // namespace
+const std::vector<ov::element::Type> netPrc = {
+    ov::element::f32,
+};
+
+INSTANTIATE_TEST_SUITE_P(ov_compiled_model,
+                         OVCompiledModelGraphUniqueNodeNamesTest,
+                         ::testing::Combine(::testing::ValuesIn(netPrc),
+                                            ::testing::Values(ov::Shape{1, 2, 5, 5}),
+                                            ::testing::Values(ov::test::utils::DEVICE_CPU)),
```

Comment:
Do not hardcode `CPU` in conformance because it can be run over any device. Please, use `targetDevice` instead of `
ov::test::utils::DEVICE_CPU`

Reply:
done